### ISSUE TITLE
Ruler Aliens are AWOL.

### ISSUE THREAD

**Issue by spankahontis** [long-war-2/lwotc#797]
Since completing "the Nest" Story mission.. I've never come across a Ruler Alien, fought around 50-70 Missions so far and never seen them once.

**Comment by wortkarg** [comment]
Grobobobo's answer from Discord:
_They aren't working correctly for now
with nest mission they don't show up on all missions except avenger defence
without it they don't show up at all because lw facilities are different than vanilla facilities_

**Comment by maldun** [comment]
This has to be newer than, there was a video pre-beta run by SG_Kaen on twitch where the rulers can be encountered [during other missions too](https://www.twitch.tv/videos/478519820?t=02h07m08s)

**Comment by spankahontis** [comment]
> 
> 
> This has to be newer than, there was a video pre-beta run by SG_Kaen on twitch where the rulers can be encountered [during other missions too](https://www.twitch.tv/videos/478519820?t=02h07m08s)

Is there a link to the new Beta version? I probably have an old version.

**Comment by maldun** [comment]
Have a look also on this issue: #279 I just found it recently, I think this explains how the current mechanic works. Seem to be a thing due to long war.
For an Overview of the releases just check them on the [releases page](https://github.com/long-war-2/lwotc/releases)

**Comment by spankahontis** [comment]
Strange thing is maldun, I just managed to get the Ruler Alien, the Beserker Queen in one of my missions just now; after 150 Missions, one finally appeared, ddn`t manage to kill it.

I'm already fully kitted out in Warden Armour and Plasma weapons, 4 Liberated Regions and i'm facing late game aliens.

True to what Kazinir stated, the Mission I encountered the Ruler Alien in was in a Troop Column Mission.
I've fought about 10 so far in this campaign and this is the first time a Ruler appeared in one.
